### PR TITLE
Updating .gitignore file

### PR DESCRIPTION
**Pull request by PalashBansal**

### CODE REVIEW

**Comment by PalashBansal** [comment]
@ihhub 
Hi, This is off the topic, I may be sounding immature.
But, I am taking part in Hacktoberfest, you must be aware of it as I can see you creating issues with labels of this. And I already have 3 PRs submitted in October, with the target of total 4.
Can you please accept 1 of the following PRs before Oct31-
1. my this PR
2. or- https://github.com/ihhub/fheroes2/pull/214
If any further changes needed, based on side effect, I will be happy to do so... (But I don't think any of the changes in above mentioned 2 PRs are risky). I am confident with both.
It will be really appreciable.

File: .gitignore [hunk 1]
```diff
@@ -30,6 +31,8 @@
 ## Ignore Visual Studio temporary files, build results, and
 ## files generated by popular Visual Studio add-ons.
 *.suo
+*.sln
```
Comment (ihhub) [inline .gitignore]: Please revert the rest of changes except this line.

Comment (PalashBansal) [reply .gitignore]: I made the changes as per your feedback. Can you please tell why .ipch and .vcxproj shouldn't be included.

Comment (ihhub) [reply .gitignore]: `.ipch` is hidden file and it's created only by solution. `vcxproj` is included into project and we change it from time to time.

**Pull request merged**